core/arch/arm/pta/se_api_self_tests.c

```diff
@@ -77,30 +77,6 @@
  * Trusted Application Entry Points
  */
 
-static TEE_Result create_ta(void)
```

Comment:
I don't understand the issue ?
Does your comment apply to all pTAs, or only to this SE API test setup?